platform/security/platform-security-session/pom.xml

```diff
@@ -27,6 +27,11 @@
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-server</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
+            <version>1.54</version>
```

Comment:
As soon as [DDF-2051](https://github.com/codice/ddf/pull/709) is merged this should be changed to `${bouncy.version}`